bot.py

```diff
@@ -40,6 +41,7 @@ def __get_process(is_last):
         pid = __get_process(args.l)
         logger.info(f'Using {"latest" if args.l else "first"} started EVE process with PID: {pid}')
 
-    bot = LocalMonitorBot(monitor_character=args.m, process_id=pid)
+    bot = LocalMonitorBot(monitor_character=args.m, process_id=pid, use_quiet_alarm=args.q)
+    logger.info('Initializing bot... This takes a few minutes so please keep your eve window open and focused')
```

Comment:
to be precise, only the first couple of seconds needs to keep the window focused. i think there is a window detection. after that the window doesnt need to be focused. but i think this log is fine atm.